edit-post/assets/stylesheets/main.scss

```diff
@@ -121,13 +121,13 @@ body.gutenberg-editor-page {
 	}
 }
 
-// Override core input styles to provide ones consistent with Gutenberg
-// @todo submit as upstream patch as well
+// Override core input styles to provide ones consistent with Gutenberg.
+// @todo Submit as upstream patch as well.
```

Comment:
Can you file an issue for this please and link to it? These TODOs will get lost to time otherwise 😄

At least if there's an issue it'll get lost to time but be tracked! 😉

Reply:
So far I've intentionally not filed, because the styles for these components were partially in flux. But as of the most recent input style change, and now this, things are solidifiying. How about this: I await a little feedback on this branch, and if the thumbs point upwards, I will create the ticket, update the PR with a link in the comment, and we'll go from there?

Reply:
Works for me. In general I say more issues (especially things like this which are easily marked as "code quality" issues we can filter out). Easier to close unneeded ones than remember long-forgotten ones 😉